### PR TITLE
Redis DB Index via Select

### PR DESCRIPTION
**Pull request by t3chguy**

### CODE REVIEW

**Comment by t3chguy** [comment]
Consider this Contribution to be MIT Licensed as of this Comment.

**Comment by DeepDiver1975** [comment]
@butonic

**Comment by Xenopathic** [comment]
:+1:

File: config/config.sample.php [hunk 1]
```diff
@@ -831,7 +831,8 @@
 'redis' => array(
 	'host' => 'localhost', // can also be a unix domain socket: '/tmp/redis.sock'
 	'port' => 6379,
-	'timeout' => 0.0
+	'timeout' => 0.0,
+	'dbindex' => 0 // Optional, if undefined SELECT will not run and will use Redis Server's default DB Index.
```
Comment (nickvergessen) [inline config/config.sample.php]: please add a trailing `,` so the next entry does not have to edit two lines again

**Comment by butonic** [comment]
@t3chguy consider this a :+1: when you add the trailing `,`, thx

**Comment by t3chguy** [comment]
Timeout didn't have one so I assumed I was conforming to your style. I'll add it as soon as I get to a computer. Thanks

**Comment by butonic** [comment]
my bad with hthe comma not being there in the first place. awsome that you clean it up ;)

**Comment by Xenopathic** [comment]
@t3chguy Could you squash your commits into one please? Thanks!

**Comment by t3chguy** [comment]
@Xenopathic Squashed

**Comment by Xenopathic** [comment]
@t3chguy That is very strange - I thought `git` ignores a line starting with a hash? Could you amend the commit message to be more descriptive? The original title was fine :smile:

**Comment by t3chguy** [comment]
I'm starting to hate `git` xD

**Comment by Xenopathic** [comment]
:+1: from before still stands

**Pull request merged**